homeassistant/components/mqtt/siren.py

```diff
@@ -171,6 +171,17 @@ class MqttSiren(MqttEntity, SirenEntity):
 
     _entity_id_format = ENTITY_ID_FORMAT
     _attributes_extra_blocked = MQTT_SIREN_ATTRIBUTES_BLOCKED
+    _attr_should_poll: bool = False
+
+    _attr_is_on: bool | None
+    _attr_extra_state_attributes: dict[str, Any]
+    _attr_name: str
```

Comment:
Did you mean to remove this line `self._attr_name = config[CONF_NAME]` ?
I think having the type hint here doesn't give us much.
 ```suggestion
```